Buildings/Controls/OBC/CDL/Reals/Ramp.mo

```diff
@@ -118,7 +118,12 @@ otherwise, <code>dy/dt = thr</code>.
 A smaller time constant <code>Td &gt; 0</code> means a higher accuracy for the derivative approximation.
 </p>
 <p>
-Note that when the input <code>activate</code> switches to <code>false</code>,
+Note that the block limits the rate of change of the input <code>u</code> in the same
+way as <a href=\"modelica://Buildings.Controls.OBC.CDL.Reals.LimitSlewRate\">
+Buildings.Controls.OBC.CDL.Reals.LimitSlewRate</a>. However, the <code>LimitSlewRate</code>
```

Comment:
@JayHuLBL : Please change the <code>....</code> to using a hyperlink as in the last sentence.
The reason is that hyperlinks can be checked automatically as part of the release checking to make sure they point to an existing model. If `LimitSlewRate` gets renamed, there would be no mechanisms that updates or checks `<code>LimitSlewRate</code>`.  This has been an issue in the past as documentation pointed to models that no longer existed.